appcues/src/main/java/com/appcues/debugger/ui/plugins/DebuggerPluginsPage.kt

```diff
@@ -0,0 +1,134 @@
+package com.appcues.debugger.ui.plugins
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyItemScope
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import com.appcues.AppcuesCustomComponentView
+import com.appcues.R
+import com.appcues.debugger.DebuggerViewModel
+import com.appcues.debugger.model.DebuggerCustomComponentItem
+import com.appcues.debugger.navigation.DebuggerRoutes.CustomComponentPage
+import com.appcues.debugger.navigation.navigateDebugger
+import com.appcues.debugger.ui.ds.DividerItem
+import com.appcues.debugger.ui.ds.FloatingBackButton
+import com.appcues.debugger.ui.ds.TextHeader
+import com.appcues.debugger.ui.ds.TextPrimary
+import com.appcues.debugger.ui.ds.TextSecondary
+import com.appcues.debugger.ui.lazyColumnScrollIndicator
+import com.appcues.debugger.ui.theme.LocalAppcuesTheme
+
+private val firstVisibleItemOffsetThreshold = 56.dp
+
+@Composable
+internal fun DebuggerPluginsPage(
```

Comment:
`Plugins` - this is really the list page for custom components right? maybe update naming to match since `Plugins` sounds possibly different